packages/@lightningjs/ui-components/src/components/Button/ButtonSmall.js

```diff
@@ -16,7 +16,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import Button from './Button.js';
+import Button from './Button';
 import * as styles from './ButtonSmall.styles.js';
```

Comment:
not related to this PR, but can we remove the .js extension from here since we're touching the file anyway